src/main/native/cpp/analysis/FilteringUtils.cpp

```diff
@@ -10,7 +10,11 @@
 #include <frc/LinearFilter.h>
 #include <frc/MedianFilter.h>
 #include <units/math.h>
+#include <wpi/StringRef.h>
 #include <wpi/math>
+#include <wpi/raw_ostream.h>
+
+#include "sysid/analysis/Storage.h"
```

Comment:
This include shouldn't be needed because it's already included by FilteringUtils.h.